handler/handler.go

```diff
@@ -79,16 +87,17 @@ func (batchHandler BatchHandler) sendToES(ctx context.Context, esDestURL string,
 	var bulkRes models.EsBulkResponse
 	if err := json.Unmarshal(jsonUpsertResponse, &bulkRes); err != nil {
 		log.Error(ctx, "error unmarshaling json", err)
+		return err
 	}
 	if bulkRes.Errors {
 		for _, resUpsertItem := range bulkRes.Items {
-			if resUpsertItem["create"].Status == 409 {
+			if resUpsertItem[create].Status == 409 {
```

Comment:
The variable name should be more explicit. Just `create` doesn't tell us what it is